pagopa/pn-frontend
# Pn 6057 fe lista notifiche pg notifiche delegate

## Short description
Fixed issue with multiple recipients column in PG notifications delegation.

## List of changes proposed in this pull request
- Fixed issue in notifications delegation

## How to test
Login in PG as DanteAlighieri, select "DivinaCommedia srl" and go to "Notifiche delegate". You should see tax IDs divided now.
